testsuites/CBLTester/System_test_multiple_device/conftest.py

```diff
@@ -389,23 +392,34 @@ def params_from_base_test_setup(params_from_base_suite_setup):
     resume_cluster = params_from_base_suite_setup["resume_cluster"]
     create_db_per_test = params_from_base_suite_setup["create_db_per_test"]
     cluster_topology = params_from_base_suite_setup["cluster_topology"]
-    # testserver_list = params_from_base_suite_setup["testserver_list"]
-    # test_name = request.node.name
```

Comment:
rather than getting from suite, can you enable this line to use test_name

Reply:
fixed